### ISSUE TITLE
Filter search box does not have a label associated to it.

### ISSUE THREAD

**Issue by chamandeep** [wenzhixin/multiple-select#138]
When filter is set to true, there is no label associated with the search box. This is not accessible to screen readers. A screen reader is a software package which converts all the text on screen into synthesized speech and is used by people who are blind or partially sighted and are not able to read print. This is one of the most accessible plugins which serve the purpose of filtering and multiple selects, so it would be great to see more accessibility features being added.

**Comment by wenzhixin** [comment]
Same with #166
